### PR TITLE
chore: update antrun plugin & fix building problem in windows

### PR DESCRIPTION
**Pull request by seagle-yuan**
update antrun plugin & fix windows os

### CODE REVIEW

File: hugegraph-dist/pom.xml [hunk 1]
```diff
@@ -178,22 +178,10 @@
...
-                                    echo
-                                </echo>
-                                <exec executable="${shell-executable}"
-                                      dir="${top.level.dir}" failonerror="true">
-                                    <arg line="./dist.sh"/>
-                                </exec>
-                            </tasks>
+                            <target>
+                                <tar
+                                    destfile="${top.level.dir}/${final.name}.tar.gz" basedir="${top.level.dir}/${final.name}" compression="gzip"/>
```
Comment (imbajin) [inline hugegraph-dist/pom.xml:183]: use one line is fine, and do u test it in local env?

Comment (seagle-yuan) [reply hugegraph-dist/pom.xml]: test on linux and windows. & compare with old and new version .

File: hugegraph-dist/pom.xml [hunk 2]
```diff
@@ -178,22 +178,9 @@
...
-                                    ${top.level.dir}/${final.name}.tar.gz"
-                                    echo
-                                </echo>
-                                <exec executable="${shell-executable}"
-                                      dir="${top.level.dir}" failonerror="true">
-                                    <arg line="./dist.sh"/>
-                                </exec>
-                            </tasks>
+                            <target>
+                                <tar destfile="${top.level.dir}/${final.name}.tar.gz" basedir="${top.level.dir}"  includes="${final.name}/*" compression="gzip"/>
```
Comment (javeme) [inline hugegraph-dist/pom.xml:182]: can wrap line for every property?

Comment (seagle-yuan) [reply hugegraph-dist/pom.xml:182]: like this
image

Comment (javeme) [reply hugegraph-dist/pom.xml:182]: yes

**Review by imbajin — APPROVED** [review]
is it compatible in macOS & win & linux? 

which OS env do u test already?

**Review by zyxxoo — APPROVED** [review]

**Pull request merged**